web/app/fr/gospeak/web/pages/published/speakers/detail.scala.html

```diff
@@ -1,12 +1,13 @@
 @import com.mohiva.play.silhouette.api.actions.UserAwareRequest
-@import fr.gospeak.core.domain.{Cfp, Group, Proposal, User}
+@import fr.gospeak.core.domain.{Cfp, Event, Group, Proposal, User}
 @import fr.gospeak.libs.scalautils.domain.Page
 @import fr.gospeak.web.auth.domain.CookieEnv
 @import fr.gospeak.web.pages.published.speakers.routes.SpeakerCtrl
 @import fr.gospeak.web.pages.published.partials.html.{container, speakerCard, talkItem, groupItem}
 @import fr.gospeak.web.domain.Breadcrumb
 
-@(speaker: User, acceptedProposals: Page[(Cfp, Proposal)], ownedGroups: Page[Group])(b: Breadcrumb)(implicit req: UserAwareRequest[CookieEnv, AnyContent], messages: Messages)
+@import java.time.LocalDateTime
+@(speaker: User, acceptedProposals: Page[(Event, Proposal)], ownedGroups: Page[Group], now: LocalDateTime)(b: Breadcrumb)(implicit req: UserAwareRequest[CookieEnv, AnyContent], messages: Messages)
```

Comment:
```suggestion
@(speaker: User, acceptedProposals: Page[(Event, Proposal)], ownedGroups: Page[Group], now: Instant)(b: Breadcrumb)(implicit req: UserAwareRequest[CookieEnv, AnyContent], messages: Messages)
```
Instant seems better (no timezone)